.cargo/config.toml

```
@@ -14,3 +14,4 @@ opt-level = 1
 
 [alias]
 build_testing = ["build", "--features", "testing"]
+neon = "run --bin neon_local --"
```

```suggestion
neon = "run --bin neon_local"
```

If it was this without the double dash, I think we'd be able to `cargo neon --release -- init` (or even `cargo neon --release init`, didn't try).

Related to: https://github.com/neondatabase/neon/pull/4240/files#r1194814187